frontend/js/src/common/brainzplayer/VolumeControlButton.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+import { useBrainzPlayerDispatch } from "./BrainzPlayerContext";
+
+function VolumeControlButton() {
+  const dispatch = useBrainzPlayerDispatch();
+  const volumeRef = React.useRef<HTMLInputElement>(null);
+  const handleVolumeChange = () => {
+    dispatch({
+      type: "VOLUME_CHANGE",
+      data: volumeRef?.current?.value ?? 100,
+    });
+  };
```

Comment:
You don't need a ref for this at all:
```suggestion
  const handleVolumeChange = (e: React.MouseEvent<HTMLInputElement>) => {
    dispatch({
      type: "VOLUME_CHANGE",
      data: e.currentTarget?.value ?? 100,
    });
  };
```

Don't forget to remove the ref prop on the input element